reliquery/storage.py

```diff
@@ -206,19 +184,22 @@ def get_all_relic_tags(self) -> List[Dict]:
 
     def get_all_relic_data(self) -> List[Dict]:
         relic_types = os.listdir(self._join_path([""]))
-
+        relic_data = []
         if relic_types:
             for relic_type in relic_types:
                 if os.path.isdir(self._join_path([relic_type])):
                     names = os.listdir(self._join_path([relic_type]))
 
                     if names:
                         for name in names:
-                            yield {
-                                "relic_name": name,
-                                "relic_type": relic_type,
-                                "storage_name": self.name,
-                            }
+                            relic_data.append(
+                                {
+                                    "relic_name": name,
+                                    "relic_type": relic_type,
+                                    "storage_name": self.name,
+                                }
+                            )
+            return relic_data
```

Comment:
the else is not needed. you can just return relic data in either case. Also, i think you can even avoid the if ```relic_types```? Wil os.listdir() ever return anything but a list?